llvm/lib/Passes/PassBuilder.cpp

```diff
@@ -1439,6 +1439,19 @@ Expected<bool> parseMachineSinkingPassOptions(StringRef Params) {
                                             "MachineSinkingPass");
 }
 
+Expected<bool> parseMachineBlockPlacementPassOptions(StringRef Params) {
+  bool AllowTailMerge = true;
+  if (Params == "no-tail-merge")
+    AllowTailMerge = false;
+  else if (!Params.empty() && Params != "enable-tail-merge")
```

Comment:
This doesn't follow the same pattern. I mean like this, no enable- https://github.com/llvm/llvm-project/blob/8c0e9adc5c1f4b1deba06d6af25c75d0c8da0557/llvm/lib/Passes/PassBuilder.cpp#L700